packages/firebase_auth/firebase_auth/ios/Classes/FLTFirebaseAuthPlugin.m

```diff
@@ -638,18 +639,37 @@ - (FIRAuth *_Nullable)getFIRAuthFromAppNameFromPigeon:(AuthPigeonFirebaseApp *)p
   return auth;
 }
 
-- (FIRAuthCredential *_Nullable)getFIRAuthCredentialFromArguments:(NSDictionary *)arguments
-                                                              app:(AuthPigeonFirebaseApp *)app {
+- (void)getFIRAuthCredentialFromArguments:(NSDictionary *)arguments
+                                      app:(AuthPigeonFirebaseApp *)app
+                               completion:(void (^)(FIRAuthCredential *credential,
+                                                    NSError *error))completion {
```

Comment:
I don't see any Pigeon related changes on the Dart part, is there some files missing?

Reply:
Yes, just double checked, it's the same error response as signing in with provider:
```bash
"[auth/operation-not-allowed] - The federated login request is not supported."
```

I'll update the code 👍

Reply:
This method is internal to the plugin and isn't part of Pigeon API.

Reply:
Oh nice didn't realise thanks :)